lib/web_ui/lib/src/engine/initialization.dart

```diff
@@ -243,22 +244,24 @@ Future<void> initializeEngineUi() async {
 AssetManager get assetManager => _assetManager!;
 AssetManager? _assetManager;
 
-Future<void> _setAssetManager(AssetManager assetManager) async {
+void _setAssetManager(AssetManager assetManager) {
   assert(assetManager != null, 'Cannot set assetManager to null');
   if (assetManager == _assetManager) {
     return;
   }
 
   _assetManager = assetManager;
+}
 
+Future<void> _downloadAssetFonts(AssetManager assetManager) async {
```

Comment:
No need to pass `assetManager` here since we already set it in `_setAssetManager`. So just do

```
await renderer.fontCollection.downloadAssetFonts(_assetManager);
```